Edit MD4CE Couey 10.22.024

solarfire9 opened the pull request:
Edit transcript. Correct minor spelling, speaker names, punctuation for clarity.

Commit:
Update v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

done to line 973

Commit:
Update v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

1:15:49 Lukach

Commit:
Update v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

Finished edit

Commit:
Merge branch 'gigaohm:main' into Edit-MD4CEI-Couey-10.22.24

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -7,16 +7,16 @@ WEBVTT
 00:22.615 --> 00:28.616
-[Charles Kovess]: We're going to have five minutes from Craig Partycooper, who's presented to us twice before.
+[Charles Kovess]: We're going to have five minutes from Craig Paardekooper, who's presented to us twice before.
 
```
Nice. I didn't know his last time.

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -625,10 +625,10 @@ WEBVTT
 
 24:01.263 --> 24:12.075
-[J Jay Couey]: That premise is central to Biology 101 at every university in the Western world, and it is absolutely central to the idea that the Human Genome Project accomplished anything at all.
+[J Jay Couey]: that premise is central to Biology 101 at every university in the Western world, and it is absolutely central to the idea that the Human Genome Project accomplished anything at all.
```
I had to double check the video for this one. Seemed odd that there was that many "that"s... but he was finding a book.

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -751,10 +751,10 @@ WEBVTT
 29:59.684 --> 30:04.147
-[J Jay Couey]: The rediscovery of Mendel's laws of heredity in the opening weeks of the 20th century.
+[J Jay Couey]: "The rediscovery of Mendel's laws of heredity in the opening weeks of the 20th century..."
 
```
Nice touch with the quotes.

solarfire9 replied:
Thanks!

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -961,13 +961,13 @@ WEBVTT
 
 40:26.062 --> 40:33.732
-[J Jay Couey]: Jeans and jeans coming on and off and how even though we can't monitor that we assume it's happening and all of this gets fueled by these.
+[J Jay Couey]: genes and genes coming on and off and how even though we can't monitor that we assume it's happening and all of this gets fueled by these.
```
W/ the way modern biology is going, there might not be any difference between jeans and genes in significance... :sweat_smile:

solarfire9 replied:
True.

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -1260,8 +1260,10 @@ WEBVTT
+
 54:38.085 --> 54:38.225
-[J Jay Couey]: of a
+[J Jay Couey]: You're going to find a lot of pushback...I would say try to get Kevin McKernan on here. Try to get Kevin McKernan on here again without me here and 
```
Wow, AI missed an entire line. Of course he was speaking a little fast, but still...

solarfire9 replied:
That surprised me too.

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -2734,10 +2736,10 @@ WEBVTT
 
 01:41:31.588 --> 01:41:34.031
-[Stephen Frost]: They're unbalanced.
+[]: Unbalanced.
```
We don't know who said this? Should we use a [?] or just leave it [ ]?

solarfire9 replied:
I checked again and it's Dave Rasnick who says "They're unbalanced."  Please change Line 2739 to read [Dave Rasnick]: They're unbalanced.

He says it twice (see Line 2742 below).

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -2860,10 +2862,10 @@ WEBVTT
 
 01:44:26.309 --> 01:44:27.510
-[Stephen Frost]: Aneuploidy.
+[David Rasnick]: Aneuploidy.
```
Wow... I don't even know what this word is and AI knows it. :sweat_smile:

solarfire9 replied:
I know, impressive. I had to look it up. My vocabulary is always expanding since watching Jay's streams!

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -3505,19 +3509,23 @@ WEBVTT
+
+[J Jay Couey]: oh wow yeah okay that's a good one um yeah...
+
+[Tom]: Well wait let me just oh yeah go ahead loop back and say that professor uh Anna Ulreich said no no this is just uh this is just crystallization and it's well documented and then after this everyone yeah why don't you comment i have maybe two more and that's it. 
```
Nice split. AI doesn't always know who's talking...

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -4033,7 +4044,8 @@ WEBVTT
 02:23:09.040 --> 02:23:09.600
-[J Jay Couey]: Is that correct?
+[
+stephen Frost]: Is that correct?
```
Probably an accidental enter here. This should be joined with the line above and start w/ a capital S.

solarfire9 replied:
Agreed.

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -4081,13 +4093,15 @@ WEBVTT
 02:25:09.278 --> 02:25:11.599
-[Stephen Frost]: Brave New World, which is his brother, right?
+[Stephen Frost]: Brave New World, 
+
```
We might have to fix this later. (e.g. merge the changes first and then make this fix so there's less changes to deal with/review)

The line above shows up in the player, but not the line below and I don't know if it's because in a VTT file it expects 1 "timestamp" and then 1 "text" line followed by a blank line as the format.

It's possible if we get rid of this space, that the lines will show up together, but if that doesn't work, then we should actually add a timestamp to the line below.

The cheat might be to use the exact same timestamp as I've seen VTT/SRT files do that and both subtitle lines will show up (e.g. overlapping speech where one comes in a little earlier and leaves a little earlier, but they overlap for a period of time).

solarfire9 replied:
I wondered about that. I wasn't sure how to add a timestamp.

solarfire9 reviewed (commented):
Please make the changes as noted.

soothspider committed:
Update v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

Small edit to speaker names.

soothspider committed:
Update v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

Attempted easy fix of no timestamps. Removing whitespace for spurious new lines to see if captions/subtitles will render as 1 block.

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -1259,7 +1259,6 @@ WEBVTT
 54:24.455 --> 54:35.483
 [J Jay Couey]: So, instead of speaking for an hour, I already probably spoke too long, I want to be able to answer as many questions as anybody wants to throw at me, even from previous talks.
-
 [Charles Kovess]: Wonderful.
```
This is what that change looks like. 
![image](https://github.com/user-attachments/assets/bc9cfa4e-956c-4533-87fc-ae299bf34c31)

soothspider commented on guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt:
```diff
@@ -3522,9 +3520,7 @@ WEBVTT
-
 [J Jay Couey]: oh wow yeah okay that's a good one um yeah...
-
 [Tom]: Well wait let me just oh yeah go ahead loop back and say that professor uh Anna Ulreich said no no this is just uh this is just crystallization and it's well documented and then after this everyone yeah why don't you comment i have maybe two more and that's it. 
```
After removing the whitespaces, it seems like any number of lines will get rendered as 1 caption.
![image](https://github.com/user-attachments/assets/f4a6a9d5-dfe8-486c-af4e-2f41f645fe10)

soothspider commented:
> Please make the changes as noted.

I was surprised I have write access to your repo and branch. 😜 

Anyhow, usually the author of the fork would make the changes so that it ends up in 1 merge.

I've added the hacky fix to the no-timestamps by bunching them together with no blank lines. That seems to be read as 1 caption by the players. A more nuanced fix can be made with SubTitle Edit (Windows) (or even online... –  https://www.nikse.dk/subtitleedit/online – but it requires having the video locally on your computer as it doesn't know how to play from a URL)

Anyhow, I'll merge the changes for now and then whenever someone has time to make better changes to the multiline captions with a better tool that lets us break lines out into different timestamps, then another PR can made for that.
